android/sdk/src/main/java/org/jitsi/meet/sdk/ProximityModule.java

```diff
@@ -74,7 +72,7 @@ public ProximityModule(ReactApplicationContext reactContext) {
             wakeLock
                 = powerManager.newWakeLock(
                         PROXIMITY_SCREEN_OFF_WAKE_LOCK,
-                        MODULE_NAME);
+                        "jitsi:"+NAME);
```

Comment:
Why is "jitsi:" needed?

Reply:
Android recommends to namespace the wakelocks, since I think they are kept in some sort of flat namespace. This generated a warning and I was close enough to see it so I fixed it.